typescript/ws-protocol/src/types.ts

```diff
@@ -71,6 +71,7 @@ export type ServerInfo = {
   name: string;
   capabilities: string[];
   supportedEncodings?: string[];
+  metadata?: Record<string, unknown>;
```

Comment:
`unknown` is fine since we only support JSON serialization. If we used a different serialization, this might need to be `<string, string>` instead. I think we can leave it how it is.

Reply:
```suggestion
  metadata?: Record<string, string>;
```